src/tests/itemCounter.test.js

```diff
@@ -0,0 +1,66 @@
+import { calculateItems } from '../resetCategoriesAndCountDishes';
+
+describe('Test de items counter from home screen', () => {
+  test('Mock an array of 6 items (dishes) and expect the function to count them (6)', () => {
+    const mockArray = [
+      {
+        strMeal: 'Beef and Mustard Pie',
+        strMealThumb: 'https://www.themealdb.com/images/media/meals/sytuqu1511553755.jpg',
+        idMeal: '52874',
+        likes: 20,
+      },
+      {
+        strMeal: 'Beef and Mustard Pie',
+        strMealThumb: 'https://www.themealdb.com/images/media/meals/sytuqu1511553755.jpg',
+        idMeal: '52874',
+        likes: 20,
+      },
+      {
+        strMeal: 'Beef and Mustard Pie',
+        strMealThumb: 'https://www.themealdb.com/images/media/meals/sytuqu1511553755.jpg',
+        idMeal: '52874',
+        likes: 20,
+      },
+      {
+        strMeal: 'Beef and Mustard Pie',
+        strMealThumb: 'https://www.themealdb.com/images/media/meals/sytuqu1511553755.jpg',
+        idMeal: '52874',
+        likes: 20,
+      },
+      {
+        strMeal: 'Beef and Mustard Pie',
+        strMealThumb: 'https://www.themealdb.com/images/media/meals/sytuqu1511553755.jpg',
+        idMeal: '52874',
+        likes: 20,
+      },
+      {
+        strMeal: 'Beef and Mustard Pie',
+        strMealThumb: 'https://www.themealdb.com/images/media/meals/sytuqu1511553755.jpg',
+        idMeal: '52874',
+        likes: 20,
+      },
+
+    ];
+
+    expect(calculateItems(mockArray)).toBe(6);
+  });
+
+  test('Mock an array of 1 item (dishes) and expect the function to count them (1)', () => {
+    const mockArray = [
+      {
+        strMeal: 'Beef and Mustard Pie',
+        strMealThumb: 'https://www.themealdb.com/images/media/meals/sytuqu1511553755.jpg',
+        idMeal: '52874',
+        likes: 20,
+      },
+    ];
+
+    expect(calculateItems(mockArray)).toBe(1);
+  });
+
+  test('Mock an array of 0 items (dishes) and expect the function to count them (0)', () => {
+    const mockArray = [];
+
+    expect(calculateItems(mockArray)).toBe(0);
+  });
+});
```

Comment:
- [ ] [OPTIONAL] You could consider using [describe.each](https://jestjs.io/docs/api#describeeachtablename-fn-timeout)